### PR TITLE
Upgrade csi-plugin-alicloud

### PR DESCRIPTION
**Pull request by ary1992**
**How to categorize this PR?**

/area open-source
/kind enhancement
/platform alicloud

**What this PR does / why we need it**:
This PR is to upgrade the version of Alicloud CSI-plugin.
**Which issue(s) this PR fixes**:
None

**Special notes for your reviewer**:

**Release note**:

```other operator
The csi-plugin-alicloud is upgraded to version v1.22.8-e84838c-aliyun
```

### CODE REVIEW

File: charts/images.yaml [hunk 1]
```diff
@@ -59,8 +59,8 @@ images:
   tag: v0.5.0
 - name: csi-plugin-alicloud
   sourceRepository: https://github.com/kubernetes-sigs/alibaba-cloud-csi-driver
-  repository: registry.eu-central-1.aliyuncs.com/acs/csi-plugin
-  tag: v1.20.5-ff6490f-aliyun
+  repository: registry.cn-hangzhou.aliyuncs.com/acs/csi-plugin
```
Comment (jia-jerry) [inline charts/images.yaml:62]:
```suggestion
  repository: registry.eu-central-1.aliyuncs.com/acs/csi-plugin
```

**Review by jia-jerry — CHANGES_REQUESTED** [review]

File: charts/images.yaml (hunk 1 above)
Comment (jia-jerry) [reply charts/images.yaml:62]: We would suggest to use images located other than Mainland China by default. You may use image-overwrite to change images when you deploy it.

**Comment by EmoinLanyu** [comment]
Can't seem to find the problems of the tests. But the PR does not affect infra part which the tests are about. I will proceed with merging.

**Review by EmoinLanyu — APPROVED** [review]
/lgtm

**Pull request merged**